### ISSUE TITLE
[Bug]: synchronization is incomplete since version 3.12.3

### ISSUE THREAD

**Issue by birnbacs** [nextcloud/desktop#6609]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

Synchronization of some items is incomplete. 
The desktop icon is green, indicating no trouble. In the app's settings, a notification is offered that a remote folder is too big for download or resides on an external medium. Both hints are incorrect: observed folder sizes were around 300 kB while a confirmation is configured for folders exceeding 500 MB. All folders on the server reside in the same folder and were synchronized in earlier versions of NC client.

### Steps to reproduce

In my case:
1.) install NC client 3.12.3
2.) have it synchronize a folder
3.) add to the folder on a different machine

### Expected behavior

* the added folder is synchronized
* should synchronization not take place, give a warning message and change the desktop icon accordingly

### Which files are affected by this bug

ZF2457

### Operating system

Mac OS

### Which version of the operating system you are running.

13.4.1 (c)

### Package

Appimage

### Nextcloud Server version

23

### Nextcloud Desktop Client version

3.12.3

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell
There is sensitive data from a patent attorney stored on the server so that a log cannot be given, sorry. However, the logs are archived and in case you need specific data please indicate what you are looking for
```

### Additional info

The problem has been observed on at least two different machines of same make and model

**Comment by birnbacs** [comment]
Desktop debug file is 97 kB, too big for upload. Please advise.

**Comment by bonojunior** [comment]
The samen happens to us (several clients) on Mac OS Ventura AND Mac OS Sonoma (latest releases).

**Comment by foranys** [comment]
Same problem on Debian 11 with appimage 3.12.3

**Comment by FrancisHGR** [comment]
Same problem on MacOS Sonoma, latest client doesn’t sync beyond 510 mb files. Always stops at this size with synchronization. Everything else is downloaded up to 510 mb.

**Comment by FrancisHGR** [comment]
got it solved by running the following commands within the docker container:

apt update && apt -y install libsmbclient-dev libmagickwand-dev && pecl install smbclient && echo “extension=smbclient.so” > /usr/local/etc/php/conf.d/docker-php-ext-smbclient.ini && pecl install inotify

Still, this bug should be fixed, ideally php-smbclient directly integrated in Nextcloud

**Comment by Rello** [comment]
Hello,

the client was enhanced significantly since the opening of this ticket.
You can help us with a retest.
If you are still encountering issues, please get back to us

**Comment by birnbacs** [comment]
Yes, the bug is still there. We are living in constant fear of missing an important sync.

**Comment by Rello** [comment]
> Yes, the bug is still there. We are living in constant fear of missing an important sync.

Please post some screeshots to show your issue with the latest version
